stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -184,11 +183,35 @@ private func configureRuntimeFromEnvironment() {
       return
     }
 
-    guard address.prefix(7) == "grpc://" else {
-      fatalError("SWIFT_TENSORFLOW_SERVER_ADDRESS must start with 'grpc://'.")
+    if let idx = address.firstIndex(of: ":") {
+      if let endIdx = address.index(idx, offsetBy: 3, limitedBy:address.endIndex) {
```

Comment:
Nested if-let chains are not great. This should use a single `if` and multiple lets separated by a comma.